tests/datasets/test_members_view.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.datasets.factories import DatasetFactory
+from vitrina.datasets.models import DatasetMembers
+from vitrina.orgs.factories import OrganizationFactory
+from vitrina.users.models import User
+
+
+@pytest.fixture
+def members_data():
+    org = OrganizationFactory(
+        title="test organization",
+        created=datetime(2022, 9, 22, 10, 30),
+        jurisdiction="test_jurisdiction",
+        kind='test_org_kind',
+        slug='test_org_slug'
+    )
+    dataset = DatasetFactory(
+        title="Testinis duomenų rinkinys",
+        title_en="Test dataset",
+        published=datetime(2022, 6, 1),
+        organization=org,
+        slug='test_dataset_slug'
+    )
+    return [dataset, org]
+
+
+@pytest.fixture
+def user():
+    user = User.objects.create_user(email="test@test.com", password="test123")
+    return user
+
+
+@pytest.mark.django_db
+def test_dataset_members_view_without_members(app: DjangoTestApp, members_data):
+    resp = app.get(reverse('dataset-members', kwargs={'org_kind': 'test_org_kind',
+                                                      'org_slug': 'test_org_slug',
+                                                      'dataset_slug': 'test_dataset_slug'}))
```

Comment:
```suggestion
def test_dataset_members_view_without_members(app: DjangoTestApp):
    dataset = DatasetFactory()
    resp = app.get(dataset.get_members_url())
```

The whole point of factories is that you don't need to create all the related objects, [factory boy](https://factoryboy.readthedocs.io/en/stable/index.html) does that for you.

Also I would suggest to add `get_members_url` method, because constructing urls with multiple arguments might lead to errors.